django_tables2_column_shifter/tests/test_base.py

```diff
@@ -12,8 +12,11 @@
     ColumnShiftTableBootstrap2,
     ColumnShiftTableBootstrap3,
     ColumnShiftTableBootstrap4,
+    ColumnShiftTableBootstrap4Responsive,
     ColumnShiftTableBootstrap5,
+    ColumnShiftTableBootstrap5Responsive
 )
+from django_tables2_column_shifter.tables import dt_version
```

Comment:
Put  this import between 11 and 12 line. 
Above package `tables` is already imported from `django_tables2_column_shifter`

Reply:
Your absolutely right!